src/dependencies.jl

```diff
@@ -106,7 +106,7 @@ end
 can_use(::Type{AptGet}) = has_apt && OS_NAME == :Linux
 package_available(p::AptGet) = can_use(AptGet) && !isempty(available_versions(p))
 function available_versions(p::AptGet)
-    vers = Compat.ASCIIString[]
```

Comment:
Ref https://github.com/JuliaLang/Compat.jl/pull/197#discussion_r62842265 and you need https://github.com/JuliaLang/Compat.jl/pull/198 to get rid of the depwarn.

Do you have evidence that the breaking of API will not affect any users of `BinDeps.jl`?